gem/templates/springster/forms/molo_form_page.html

```diff
@@ -92,7 +92,8 @@ <h6 class="surveys__pagination-title">
 
   {# START FORM #}
     {% if form %}
-    <form action="{% if form.display_form_directly %}{% pageurl form %}{% else %}{% pageurl self.specific %}{% endif %}{% if self.multi_step or self.has_page_breaks %}?p={{ fields_step.number|add:"1" }}{% endif %}" method="post" class="{% if self.display_form_directly %}polls{% else %}surveys{% endif %}__form {% if self.display_form_directly %}polls{% else %}surveys{% endif %}__form{{self.get_effective_extra_style_hints}}" novalidate>
+    ==>{% pageurl self.specific %}
```

Comment:
`==>{% pageurl self.specific %}` shouldn't it be `{% pageurl self.specific %}` perhaps